code/dynamic_programming/matrix_chain_multiplication/MCM.java

```diff
@@ -0,0 +1,91 @@
+//cosmos
```

Comment:
Wrong comment. Kindly review guidelines 👍

Reply:
Can you please tell what comment is to be added instead of this on the top  of the code! :)